cmd/ipfs-cluster-follow/main.go

```diff
@@ -0,0 +1,286 @@
+// The ipfs-cluster-follow application.
+package main
+
+import (
+	"fmt"
+	"os"
+	"os/user"
+	"path/filepath"
+
+	"github.com/ipfs/ipfs-cluster/api/rest/client"
+	"github.com/ipfs/ipfs-cluster/cmdutils"
+	"github.com/ipfs/ipfs-cluster/version"
+	"github.com/multiformats/go-multiaddr"
+	"github.com/pkg/errors"
+
+	semver "github.com/blang/semver"
+	logging "github.com/ipfs/go-log"
+	cli "github.com/urfave/cli/v2"
+)
+
+const (
+	// ProgramName of this application
+	programName     = "ipfs-cluster-follow"
+	clusterNameFlag = "clusterName"
+	logLevel        = "info"
+)
+
+// We store a commit id here
+var commit string
+
+// Description provides a short summary of the functionality of this tool
+var Description = fmt.Sprintf(`
+%s helps running IPFS Cluster follower peers.
+
+Follower peers subscribe to a Cluster controlled by a set of "trusted
+peers". They collaborate in pinning items as dictated by the trusted peers and
+do not have the power to make Cluster-wide modifications to the pinset.
+
+Follower peers cannot access information nor trigger actions in other peers.
+
+%s can be used to follow different clusters by launching it 
+with different options. Each Cluster has an identity, a configuration 
+and a datastore associated to it, which are kept under 
+"~/%s/<cluster_name>".
+
+For feedback, bug reports or any additional information, visit
+https://github.com/ipfs/ipfs-cluster.
+
+
+EXAMPLES:
+
+List configured follower peers:
+
+$ %s
+
+Display information for a follower peer:
+
+$ %s <clusterName> info
+
+Initialize a follower peer:
+
+$ %s <clusterName> init <example.url>
+
+Launch a follower peer (will stay running):
+
+$ %s <clusterName> run
+
+List items in the pinset for a given cluster:
+
+$ %s <clusterName> list
+
+Getting help and usage info:
+
+$ %s --help
+$ %s <clusterName> --help
+$ %s <clusterName> info --help
+$ %s <clusterName> init --help
+$ %s <clusterName> run --help
+$ %s <clusterName> list --help
+
+`,
+	programName,
+	programName,
+	DefaultFolder,
+	programName,
+	programName,
+	programName,
+	programName,
+	programName,
+	programName,
+	programName,
+	programName,
+	programName,
+	programName,
+	programName,
+)
+
+var logger = logging.Logger("clusterfollow")
+
+// Default location for the configurations and data
+var (
+	// DefaultFolder is the name of the cluster folder
+	DefaultFolder = ".ipfs-cluster-follow"
```

Comment:
Would be nice to have someone test how this plays on windows for a default folder.